drivers/sensor/mcp9600/mcp9600_trigger.c

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2020 SER Consulting LLC, Steven Riedl
+ * Copyright (c) 2019 Peter Bigot Consulting, LLC
+ *
+ * !!!! NOT DONE OR TESTED, Converted from mcp9808
```

Comment:
This does not inspire confidence....

Reply:
Missed that, was actually well tested, even ran some fake value insertion tests to insure the negative temps were correct, will remove...

Reply:
Actually trigger was not tested, just removed all trigger functionality. Can be added back in at a later time properly as the device has 4 triggers.